src/sentry/deletions/defaults/project.py

```diff
@@ -98,6 +100,13 @@ def get_child_relations(self, instance: Project) -> list[BaseRelation]:
                 {"snuba_query__subscriptions__project": instance},
             )
         )
+        relations.append(
+            ModelRelation(
+                Rule,
+                {"project_id": instance.id},
+                RuleDeletionTask,
+            )
+        )
```

Comment:
the project deletion task isn't picking up the RuleDeletionTask without adding it explicitly

Reply:
maybe it will work without declaring RuleDeletionTask